scout/adapter/mongo/event.py

```diff
@@ -294,7 +294,7 @@ def add_phenotype(
         if hpo_term:  # User specified an HPO term
             hpo_results = [hpo_term]
         elif omim_term:  # User specified an OMIM diagnosys, collect its associated HPO terms
-            disease_obj = self.disease_term(omim_term)
+            disease_obj = self.disease_term(omim_term, filter_project={"hpo_terms": 1})
```

Comment:
👍 That should do it.

Reply:
Also note in the docstring. That is a slightly unexpected construction, isn't it? 😊